assets/js/app/llms-forms.js

```diff
@@ -378,12 +377,48 @@ LLMS.Forms = {
 			this.$states.html( '<option>&nbsp</option>' );
 			this.disable_field( this.$states );
 		} else {
+			var placeholder = this.get_state_placeholder( this.$states, country_code );
 			this.enable_field( this.$states );
 			this.$states.html( opts );
+			if ( '' !== placeholder ) {
+				this.set_state_placeholder( this.$states, placeholder );
+			}
 		}
 
 	},
 
+    /**
+	 * Get the state placeholder and/or {REGION}
+	 *
+	 * @since [version]
+	 *
+	 * @param {String} Creates the default text from the text that is in the state/region field in the wordpress admin
+	 * @return {String} Returns the placeholder text from the admin area to the state field.
+	 * @return {String} Returns the {REGION} text from the admin area to the state field if nothing is found Default is used.
+	 * @return {void}
+	 */
+	 get_state_placeholder: function( holder, country_code ) {
+		var placeholder;
+		if (typeof this.address_info[country_code] === 'undefined' ) {
+		placeholder = holder.attr( 'placeholder' ).replace( '{REGION}', LLMS.l10n.translate( "DEFAULT" ));
+		} else {
+		placeholder = holder.attr( 'placeholder' ).replace( '{REGION}', this.address_info[country_code].state );
+		}
+		return placeholder;
+		},
+	
+	/**
+	 * Set the state placeholder to the state field
+	 *
+	 * @since [version]
+	 *
```

Comment:
```suggestion
		/**
	 * Set the state placeholder to the state field
	 *
	 * @since [version]
	 * @param  {} holder
	 * @param  {} placeholder
	 * @return {}
	 */
```